opendevin/sandbox/docker/ssh_box.py

```diff
@@ -43,7 +45,7 @@
     USER_ID = os.getuid()
 
 
-class DockerSSHBox(Sandbox):
+class DockerSSHBox(Sandbox, PluginMixin):
```

Comment:
Why is `PluginMixin` just on the `DockerSSHBox`? Can't we just add it to the root `Sandbox` class, so that every sandbox gets it for free?

Reply:
That's a great suggestion! I have not done this yet because I'm unsure whether adding mixin to the abstract base class is good.

But based on your suggestion of refactoring out the `chown` to separate function, I guess maybe a better route would be we keep the `Sandbox` as an interface and starts a `BaseSanbox` to have some default implementation of these and mixin the `PluginMixin` (and reduce code duplication).

Reply:
Well, [looks like](https://stackoverflow.com/questions/53213543/is-it-a-good-practice-to-put-common-methods-to-an-abstract-class-in-python) having non-abstract methods in the abstract class is fine. I just update the PR to include this mixin in the base class.